package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bellese/angular-design-system",
-  "version": "2.2.24",
+  "version": "2.2.25",
```

Comment:
Does the P/R require an update to the version. We might want to consider setting up development branch and pushing releases to master instead of versioning every time there is an update.

Reply:
This is something that automatically gets updated when we release a new version to npmjs.org.  It can be ignored for now, and I'll look into a better workflow in the future.